core-analysis/01-filter-sce.R

```diff
@@ -196,6 +196,7 @@ if (opt$filtering_method == "manual") {
                                         mito_percent_cutoff = opt$mito_percent_cutoff,
                                         detected_gene_cutoff = opt$detected_gene_cutoff,
```

Comment:
```suggestion
                                        min_gene_cutoff = opt$min_gene_cutoff,
```
I also think this should be changed to `min_gene_cutoff` to match the change in argument name.